e2etest/newe2e_task_resourcemanagement.go

```diff
@@ -220,3 +220,26 @@ func ValidateErrorOutput(a Asserter, stdout AzCopyStdout, errorMsg string) {
 	fmt.Println(stdout.String())
 	a.Error("expected error message not found in azcopy output")
 }
+
+func ValidateMultipleErrors(a Asserter, stdout AzCopyStdout, errorMsg []string) {
```

Comment:
Q: Is this function designed to

- Check all errors in the list are present
or
- Check at least one error in the list is present

Reply:
This is designed to check that at least one error is present. I can rename it to ValidateContainsError or something of the sort.